src/cascadia/WindowsTerminal/WindowThread.cpp

```diff
@@ -198,6 +182,11 @@ int WindowThread::_messagePump()
 
     while (GetMessageW(&message, nullptr, 0, 0))
     {
+        if (message.message == AppHost::WM_REFRIGERATE)

```

Comment:
```suggestion
        // We're using a single window message (WM_REFRIGERATE) to indicate both
        // state transitions. In this case, the window is actually being
        // refrigerated. This will break us out of our main message loop, and if
        // we're on windows 10, we'll eventually start the loop in
        // WindowThread::KeepWarm to await being microwaved
        if (message.message == AppHost::WM_REFRIGERATE)
```